### PR TITLE
Update Encrypt.php

### PR DESCRIPTION
**Pull request by sarciszewski**
Read this please:
http://phpsecurity.readthedocs.org/en/latest/Insufficient-Entropy-For-Random-Values.html

Also, this should probably be wrapped inside of an authentication mechanism (Encrypt-then-HMAC) if it isn't already employed elsewhere.

### CODE REVIEW

File: classes/Kohana/Encrypt.php [hunk 1]
```diff
@@ -162,6 +162,9 @@ public function encode($data)
 			// The system random number generator must always be seeded each
 			// time it is used, or it will not produce true random results
 			mt_srand();
```
Comment (shadowhand) [inline classes/Kohana/Encrypt.php:164]: This should be fixed, [mcrypt_create_iv docs](http://php.net/manual/en/function.mcrypt-create-iv.php) state that using srand() is no longer required when using MCRYPT_RAND. These lines should be removed. I'm not sure where using mt_srand() came from.

Comment (sarciszewski) [inline classes/Kohana/Encrypt.php:164]: Well, if there's mystery code then this merits further examination.

**Comment by shadowhand** [comment]
This is awesome, but it is insufficient. Since PHP 5.3, additional modes are available in Windows, which means that we can just error out when neither of the other modes are available.

**Comment by shadowhand** [comment]
And finally, this pull request is targeting the wrong branch. All pull requests must be made against develop branch, not master.

**Comment by sarciszewski** [comment]
My mistake.

**Comment by zombor** [comment]
Please see the contribution documentation here: https://github.com/kohana/kohana/blob/3.3/master/CONTRIBUTING.md

**Pull request closed without merging**